src/utils/utils.tsx

```diff
@@ -25,20 +27,36 @@ export const getGeography = (city: string[] | null | undefined, state: string[]
     return Translate("Not Reported");
   }
 
-  function renderOutGeography(geo: string[] | null | undefined) {
+  function renderOutGeography(geo: string[] | string | null | undefined) {
     if (!geo || geo.length === 0) {
       return undefined
     }
-    else if (geo.length > 1 && typeof geo !== "string") {
-      return geo?.join(", ");
+    if (typeof geo === "string"){
+      return geo.trim() + ", "
+    }
+    else if (geo.length > 1) {
+
+      return geo.join(", ") + ", ";
     }
-    return geo + ", "
+    return geo[0].trim() + ", "
+
   }
 
   //return  [renderOutGeography(city), renderOutGeography(state), country].filter(Boolean).map((str)=>{return (<React.Fragment>{str}</React.Fragment>)})
   return "" + (renderOutGeography(city) ? renderOutGeography(city) : "") + (renderOutGeography(state) ? renderOutGeography(state) : "")  + country

```

Comment:
Couldn't we just make renderOutGeography return "" if the input is null/empty? Then we could remove these ternaries which make the code look a bit confusing